docs/sources/variables/chained-variables.md

```diff
@@ -0,0 +1,174 @@
+weight = 600
++++
+
+# Chained variables
+
+_Chained variables_, also called _linked variables_ or _nested variables_, are query variables with one or more other variables in their variable query. This page explains how chained variables work and provides links to example dashboards that use chained variables.
+
+Chained variable queries are different for every data source, but the premise is the same for all. You can use chained variable queries in any data source that allows them.
+
+Extremely complex linked templated dashboards are possible, 5 or 10 levels deep. Technically, there is no limit to how deep or complex you can go, but the more links you have, the greater the query load.
+
+## Grafana Play dashboard examples
+
+The following Grafana Play dashboards contain fairly simple chained variables, only two layers deep. To view the variables and their settings, click **Dashboard settings** (gear icon) and then click **Variables**. Both examples are expanded in the following section.
+
+- [Graphite Templated Nested](https://play.grafana.org/d/000000056/graphite-templated-nested?orgId=1&var-app=country&var-server=All&var-interval=1h)
+- [InfluxDB Templated](https://play.grafana.org/d/000000002/influxdb-templated?orgId=1)
+
+## Examples explained
+
+Variables are useful to reuse dashboards, dynamically change what is shown in dashboards. Chained variables are especially useful to filter what you see.
+
+Create parent/child relationship in variable, sort of a tree structure where you can select different levels of filters.
+
+The following sections explain the linked examples in the dashboards above in depth and builds on them. While the examples are data source-specific, the concepts can be applied broadly.
+
+### Graphite example
+
+In this example, you have several applications. Each application has a different subset of servers. It is based on the [Graphite Templated Nested](https://play.grafana.org/d/000000056/graphite-templated-nested?orgId=1&var-app=country&var-server=All&var-interval=1h).
+
+Now, you could make separate variables for each metric source, but then you have to know which server goes with which app. A better solution is to use one variable to filter another. In this example, when the user changes the value of the `app` variable, it changes the dropdown options returned by the `server` variable. Both variables use the **Multi-value** option and **Include all option**, allowing users to select some or all options presented at any time.
+
+#### app variable
+
+The query for this variable basically says, "Give me all the applications that exist."
+
+```
+apps.*
+```
+
+The values returned are `backend`, `country`, `fakesite`, and `All`.
+
+#### server variable
+
+The query for this variable basically says, "Give me all servers for the currently chosen application." 
+
+```
+apps.$app.*
+```
+
+By itself, this query is nonsense, because `$app` has no value. However, if the user selects `backend`, then the query changes to:
```

Comment:
I wouldn't call it nonsense because the query will be `apps.{backend,country,fakesite}.*` and can also be useful depending on use case. For example, if you only want to filter on server, but not app given that you have same server value for multiple apps.